payments/src/lib.rs

```diff
@@ -0,0 +1,601 @@
+#![allow(clippy::unused_unit, unused_qualifications, missing_debug_implementations)]
```

Comment:
can you add crate docs, just copy the readme will be good

Reply:
are those allow absolutely necessary? some comments to explain them will be helpful